pyproject.toml

```diff
@@ -174,28 +298,27 @@ docs-serve = "sphinx-autobuild docs docs/_build/ -j auto --watch litestar --watc
 lint = "pre-commit run --all-files"
 test = "pytest tests docs/examples"
 
-[build-system]
-build-backend = "hatchling.build"
-requires = ["hatchling"]
-
-[tool.codespell]
-ignore-words-list = "selectin"
-skip = 'pdm.lock,docs/examples/contrib/sqlalchemy/us_state_lookup.json'
-
-[tool.coverage.run]
-concurrency = ["multiprocessing", "thread"]
-omit = ["*/tests/*", "*/litestar/plugins/sqlalchemy.py"]
-parallel = true
-plugins = ["covdefaults"]
-source = ["litestar"]
+[tool.pydantic-mypy]
+init_forbid_extra = true
+init_typed = true
+warn_required_dynamic_aliases = true
+warn_untyped_fields = true
 
-[tool.coverage.report]
-exclude_lines = [
-  'except ImportError\b',
-  'if VERSION.startswith("1"):',
-  'if pydantic.VERSION.startswith("1"):',
+[tool.pyright]
+disableBytesTypePromotions = true
+exclude = [
+  "test_apps",
+  "tools",
+  "docs",
+  "tests/examples",
+  "tests/docker_service_fixtures.py",
+  "litestar/contrib/pydantic/pydantic_dto_factory.py",
+  "litestar/contrib/pydantic/pydantic_init_plugin.py",
+  "tests/unit/test_contrib/test_sqlalchemy.py"
 ]
-fail_under = 96
```

Comment:
is this thing on crack it just moves keys out from under sections to random ones